src/main/java/server/HTTProtocol.java

```diff
@@ -23,8 +23,8 @@ public HTTProtocol(Client client) {
     public void run() {
         Request request = RequestParser.parse(in);
         Response response = ResponseBuilder.buildResponse(request);
-
-        out.print(ResponseParser.parse(response));
+        String parsedResponse = ResponseHTTPFormatter.formatHTTPResponse(response);
```

Comment:
Small thing, but the naming here is a bit redundant. `ResponseHttpFormatter.format` conveys about the same information, imo.